### ISSUE TITLE
Should text content be used for aria-valuetext?

### ISSUE THREAD

**Issue by JAWS-test** [w3c/aria#1308]
Should the text content of an element that aria-valuetext may own but does not own be used as value for aria-valuetext, e.g.

```
<meter>3</meter>
<div role=meter>3</div>
```

Attention: In the example further required attributes are missing. But the question is whether the 3 should be interpreted as aria-valuetext. If not, this should perhaps be mentioned explicitly in the specification, so that, for example, in the first [WHATWG example](https://html.spec.whatwg.org/multipage/form-elements.html#the-meter-element) ("Instead, one would either not include the meter element, ..."), the assumption does not arise that the text content would be perceivable with AT. 

See also: https://github.com/FreedomScientific/VFO-standards-support/issues/419

**Comment by aleventhal** [comment]
I think in the case of ```<meter>```, yes it should be, as the inner text is a fallback description.
However, in the case of the ARIA meter role, I think no, because that inner contents is likely the implementation of the meter appearance. The author should be able to use aria-valuetext with the role meter.

**Comment by a11ydoer** [comment]
More info about [aria-valuetext in meter](https://w3c.github.io/aria-practices/#range_related_properties_using_aria-valuetext)

"When the element's values are contained within a range but those values are not numeric (such as "small", "medium" and "large"), or they are numeric but there is value in communicating more information than just a number, **aria-valuetext** is used to surface the text value to assistive technologies. **Only use aria-valuetext when aria-valuenow is not sufficiently meaningful for users because using aria-valuetext will prevent assistive technologies from communicating aria-valuenow.**

**Comment by jnurthen** [comment]
I'm not sure there is anything to do here. Anyone disagree?
